recipes/chafa/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "chafa" %}
+{% set version = "1.14.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpjansson/{{ name }}/releases/download/{{ version }}/chafa-{{ version }}.tar.xz
+  sha256: 7b5b384d5fb76a641d00af0626ed2115fb255ea371d9bef11f8500286a7b09e5
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - pkg-config
+    - make
+  host:
+    - freetype
+    - cairo
+    - glib
+    - libjpeg-turbo
+    - librsvg
+    - libtiff
+    - libwebp
+    - libavif
+    - libjxl
+    - libpng  # [linux]
+    - zlib  # [linux]
+    - gdk-pixbuf  # [osx]
+    - gettext  # [osx]
+  run:
+    - freetype
+    - cairo
+    - glib
+    - libjpeg-turbo
+    - librsvg
+    - libtiff
+    - libwebp
+    - libavif
+    - libjxl
+    - libpng  # [linux]
+    - zlib  # [linux]
+    - gdk-pixbuf  # [osx]
+    - gettext  # [osx]
+
+test:
+  commands:
+    - chafa --version
+
+about:
+  home: https://hpjansson.org/chafa
+  summary: Versatile and fast Unicode/ASCII/ANSI graphics renderer
+  license: LGPL-3.0-or-later
+  license_family: LGPL
+  license_file: COPYING
+  dev_url: https://github.com/hpjansson/chafa
+
+extra:
+  recipe-maintainers:
+    - pavelzw
```

Comment:
@eunos-1128 @peterjc would you like to be maintainers as well for this package?

Reply:
I'm willing, but a bit out of my comfort zone here (Python packaging is safer ground).

Reply:
I'm fine either way:)